config/lava/iec-security/iec-security.jinja2

```diff
@@ -0,0 +1,13 @@
+- test:
+    name: {{ plan }}
+    description: "cyber security standard tests such as IEC-62443-4-2"
+    timeout:
+      minutes: 20
+    definitions:
+    - repository: https://github.com/kernelci/iec-security.git
+      from: git
+      branch: kernelci.org
+      path: iec-security-tests/Singlenode-TestDefinition.yaml
```

Comment:
The jobs are all failing to execute `runTest.sh`, it looks like there's a problem in the YAML file:
https://lava.collabora.co.uk/scheduler/job/4882291#L2444

```
cd /lava-4882291/0/tests/0_iec-security/iec-security-tests/singlenode-testcases/
./run_all.sh
```
This looks OK, but then:
```
TC_CR1.11_1
./run_all.sh: eval: line 1: ./runTest.sh: not found
```

There's also this problem:
```
TC_CR1.3_1
preTest: TC_CR1.3_1 : Validating Creating human user accounts
./runTest.sh: line 34: dpkg: not found
postTest: TC_CR1.3_1 : Validating Creating human user accounts
INFO: check_user: 'user1' doesn't exist
./runTest.sh: line 110: getent: not found
```

The test plan is currently set up to use `buildroot`, now it looks like the `iec` tests are more expecting a Debian system with `dpkg` installed.  So using the `buster-nfs` root file system instead would probably be a better fit.

Reply:
I will try to use the CIP rootfs after the cip-core-image pull request get merged https://github.com/kernelci/kernelci-core/pull/790